static/scripts/sc-blog.js

```diff
@@ -0,0 +1,29 @@
+const stripHtml = require('string-strip-html');
+
+function fetchContent() {
+	$('.sc-blog .spinner').show();
+	$('.sc-blog .placeholder').hide();
+
+	$.ajax({
+		url: '/ghost/startseite-hpi-schul-cloud',
```

Comment:
make this optional, add something like GHOST_STARTPAGE_SLUG and add this only if defined, using this we can reuse it somewhere else.